CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 1.9.6
+
+* Revert #597 (#639)
+* Harden check when rendering media library (#652)
+* Clean comment cache after direct SQL queries. (#658)
+* REST API routes require a permission_callback (#669)
+* Load CPT support later to avoid fatals with early use of WP_Query (#672)
+
+props [anigeluk], [david-binda], [GaryJones], [jeffersonrabb], [mslinnea], [philipjohn], [rebeccahum]
```

Comment:
`anigeluk`, `jeffersonrabb`, and `mslinnea` all need corresponding links at the bottom of the file.

(There are also a couple of out-of-alphabetical-order ones at the very bottom - please move them to the correct order as well)